data/events/2020-guadalajara.yml

```diff
@@ -0,0 +1,88 @@
+name: "2020-guadalajara" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2020" # The year of the event. Make sure it is in quotes.
+city: "Guadalajara" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdaysgdl" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Guadalajara!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+startdate: 2020-02-20T08:00:00-06:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  2020-04-21T08:00:00-06:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```

Comment:
We updated the date, although the month wasn't updated. Let's change it to `enddate:  2020-02-21T18:00:00-06:00` (month 02, hour 18)

Reply:
Date was updated to 18 CST and also the month was updated